internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go

```diff
@@ -163,16 +163,14 @@ func (m *DynamicScaleDownManager) createAMapOfProcessedClusters(kafkaStreamingUn
 	var processedClusters map[string]processed = map[string]processed{}
 
 	for i, kafkaStreamingUnitCountPerCluster := range kafkaStreamingUnitCountPerClusterList {
-		if kafkaStreamingUnitCountPerCluster.ClusterType != api.ManagedDataPlaneClusterType.String() {
```

Comment:
Why is this removed?

Reply:
We needed to construct the struct needed for `ShouldScaleDown` evaluation for enterprise clusters. 
With this check, all enterprise clusters were skipped from that struct - which had another ripple effect of not considering enterprise clusters as being processed in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/aedf21a02b82289bff00df4cd337fd83ec6d2d26/internal/kafka/internal/workers/cluster_mgrs/dynamic_scale_down_mgr.go#L92